### ISSUE TITLE
Support Turkish lira

### ISSUE THREAD

**Issue by mrahmadt** [kennedyoliveira/alfred-rates#6]
Hello

Could you please include Turkish lira in your workflow? or maybe to have a HOWTO document so I can include it

Thanks

**Comment by kennedyoliveira** [comment]
Hello man!

I added the support for you! Sorry for no documentation of how to add new currencies, its pretty simple, just add a new line in the currencies.csv as the following:

Id,Name,Code,Simbol,Country,Flag
Next Id Number, Name of the Currency, Code of the currency, Simbol of the currency, Country of the Currency, Name of the Flag file of the currency
154,Turkish Lira,TRY,₺,Turkey,turkey.png

I did't document because a plan to change it.

Anyway, update your workflow and run rateclear to clear the caches and reload the new currency!
